pkg/docker/docker.go

```diff
@@ -106,26 +120,25 @@ type Docker interface {
 // Client contains all methods called on the go Docker
 // client.
```

Comment:
yes it is.  For Client:

`Client contains all methods used when interacting directly with docker engine-api instead of the k8s abstraction around docker engine-api`

I also updated the comment around the Docker interface:

`Docker is the interface between STI and the k8s abstraction around docker engine-api.`